### PR TITLE
Start phase 2 and example decoding

### PR DESCRIPTION
**Pull request by craigp-atw**
- Render the Phase 2 encoding from the examples as simple text (for now).
- Show a readable version of the arguments associated with each example in the verse-selection dropdown.
- Show all occurrences of a word within a verse in the verse-selection dropdown.

**Before: Verse Selection Dropdown for the Adjective good-A**
image

**After: Verse Selection Dropdown for the Adjective good-A**
image

**After: Phase 2 rendering of Genesis 2:9 as an example of good-A**
image

Feel free to contact me with any questions about these changes!

### CODE REVIEW

**Comment by craigp-atw** [comment]
Some helpful words in the Ontology for viewing the changes:
- person-A - a noun with many semantic roles
- take-B and speak-B - verbs with many semantic role arguments
- be-V - verb with an agent proposition
- able-A - adjective with a patient proposition
- beautiful-A - comparative form in Judges 15:2
- also-A, also-B, also-C - adverbs with different usages
- on-A - adposition in an adjunct phrase
- -Kinship - adposition in a Noun-Noun relationship
- before-A - adposition in an adverbial clause

File: app/src/lib/server/index.d.ts [hunk 1]
```diff
@@ -42,10 +42,30 @@ type Reference = {
...
+}
+
+type SourceEntity = {
+	label: string
+    features: string
+	sense: string
+    entity: string
+}
+
+type SemanticRepresentation = SourcsEntity[]
```
Comment (longrunningprocess) [inline app/src/lib/server/index.d.ts:71]:
```suggestion
type SemanticRepresentation = SourceEntity[]
```

**Review by longrunningprocess — COMMENTED** [review]
sone pending changes from our pairing session...

File: app/src/lib/Examples.js [hunk 2]
```diff
@@ -0,0 +1,188 @@
+
+// TODO: maybe this belongs in the data layer...need to consider
+/**
+ * Transforms a list of examples with "flat" References into a nested structure
+ *
+ *	{
+ *		{											{
```
Comment (craigp-atw) [inline app/src/lib/Examples.js:7]: The formatting looks fine in VSCode, not sure why it's so messed up...

**Review by longrunningprocess — APPROVED** [review]

**Pull request merged**